nuklear.h

```diff
@@ -17239,10 +17239,14 @@ nk_font_bake_convert(void *out_memory, int img_width, int img_height,
     NK_ASSERT(img_height);
     if (!out_memory || !in_memory || !img_height || !img_width) return;
 
-    dst = (nk_rune*)out_memory;
+    dst = (nk_byte*)out_memory;
     src = (const nk_byte*)in_memory;
-    for (n = (int)(img_width * img_height); n > 0; n--)
-        *dst++ = ((nk_rune)(*src++) << 24) | 0x00FFFFFF;
+    for (n = (int)(img_width * img_height); n > 0; n--) {
+        *dst++ = 0xff;  /* r */
+        *dst++ = 0xff;  /* g */
+        *dst++ = 0xff;  /* b */
```

Comment:
It might be faster to call `memset(out_memory, 0xff, img_width * img_height)` first to set the R/G/B components, particularly on architectures without unaligned memory access that wouldn't be able to optimise this to use 32-bit writes like the old version did.